packages/block-library/src/gallery/use-short-code-transform.js

```diff
@@ -0,0 +1,43 @@
+/**
+ * External dependencies
+ */
+import { every } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { useSelect } from '@wordpress/data';
+
+export default function useShortCodeTransform( shortCodeTransforms ) {
+	const newImageData = useSelect(
+		( select ) => {
+			if ( ! shortCodeTransforms || shortCodeTransforms.length === 0 ) {
+				return;
+			}
+			const getMedia = select( 'core' ).getMedia;
```

Comment:
This (and some other places) should be updated to use the new way of selecting store data:

```js
import { store as coreStore } from '@wordpress/core-data';

// ...

const { getMedia } = select( coreStore );
```

Reply:
I think I have them all sorted now, were just two that I could see that I had missed.